README.md

```diff
@@ -1,3 +1,166 @@
 # Cluster Node Tuning Operator
 
-The Cluster Node Tuning Operator manages cluster node-level tuning for [OpenShift](https://openshift.io/).
+The Cluster Node Tuning Operator manages cluster node-level tuning for
+[OpenShift](https://openshift.io/).
+
+The majority of high-performance applications require some
+level of kernel tuning.  The operator provides a unified
+management interface to users of node-level sysctls and more
+flexibility to add [custom tuning](#custom-tuning-specification)
+specified by user needs.  The operator manages containerized
+[tuned](https://github.com/redhat-performance/tuned/)
+daemon for [OpenShift](https://openshift.io/) as
+a Kubernetes DaemonSet.  It ensures [custom tuning
+specification](#custom-tuning-specification) is passed to all
+[containerized tuned](https://github.com/openshift/openshift-tuned)
+daemons running in the cluster in the format they understand.
```

Comment:
who is "they"? If it's the daemons, we can just say "the format that the daemons understand."